juju/juju
# [JUJU-4307] Injects ctrlConfigService in agentconfigupdater worker

This commit injects ctrlConfigService in agentconfigupdater worker and fixes unit-tests.

## Checklist

*If an item is not applicable, use `~strikethrough~`.*


## QA steps

*Commands to run to verify that the change works.*

```sh
go test github.com/juju/juju/worker/agentconfigupdater/... -gocheck.v 
```
